### ISSUE TITLE
formatting of `rasa data validate` output

### ISSUE THREAD

**Issue by dcsan** [RasaHQ/rasa#5391]
**Description of Problem**:
the new `rasa data validate` but it prints the info in a hard to parse format:

![image](https://user-images.githubusercontent.com/514002/76207350-da26ab00-61ba-11ea-89f0-ea1c7c688a7f.png)

I have to read the essay and parse it in detail then find the elements to change.

**Overview of the Solution**:
It would be awesome to get a simpler output like

```
validation results
  file: nlu.md
  missing intents:

  ## intent:thanks
  --

```

if you can point me in the right place to start maybe I can try a PR on this

**Comment by dcsan** [comment]
ideally at the end it could even `autofix` some of these by at least putting in placeholder data

```
add placeholder fixes? [y/N]

added ##intent: thanks
```

where it might add:
```
## intent: thanks
- PLACEHOLDER 1
- PLACEHOLDER 2
```

this is easy to then scan for and replace later.

**Comment by sara-tagger** [comment]
Thanks for the issue, @rctatman will get back to you about it soon!

###### You may find help in the [docs](https://rasa.com/docs/) and the [forum,](https://forum.rasa.com/) too 🤗
